source/_integrations/ffmpeg_motion.markdown

```diff
@@ -18,7 +18,7 @@ The `ffmpeg` {% term integration %} allows you to use any video feed with [FFmpe
 
 {% note %}
 
-If the `ffmpeg` process is broken, the sensor will be unavailable. To control the FFmpeg process of sensor, use the service `ffmpeg.start`, `ffmpeg.stop`, `ffmpeg.restart`.
+If the `ffmpeg` process is broken, the sensor will be unavailable. To control the FFmpeg process of the sensor, use the `ffmpeg.start`, `ffmpeg.stop`, and `ffmpeg.restart` actions.
```

Comment:
```suggestion
If the `ffmpeg` process is broken, the sensor will be unavailable. To control the FFmpeg process of the sensor, use the `ffmpeg.start`, `ffmpeg.stop` and `ffmpeg.restart` actions.
```

Reply:
This suggestion is not correct, it should have a comma here.

Reply:
now I learned about the so called "Oxford Comma" ... it is still looks weird to me :see_no_evil: but i'm not a native English :grin: